Applications/YOLO/jni/main.cpp

```diff
@@ -217,7 +215,8 @@ int main(int argc, char *argv[]) {
   // create train and validation data
   std::array<UserDataType, 2> user_datas;
   try {
-    user_datas = createFakeDataGenerator(batch_size, data_size, data_split);
+    user_datas =
+      createDetDataGenerator("./train_dir/", "./valid_dir/", 5, 3, 416, 416);
```

Comment:
Please make this values like image height/width size as constant variable

Reply:
I made it using constant variables. thanks!